apple/internal/partials/framework_import.bzl

```
@@ -292,31 +298,28 @@ def _generate_symbols(
     )
     outputs = [symbols_dir]
 
-    commands = ["mkdir -p \"${OUTPUT_DIR}\""]
+    args = actions.args()
+    args.add("--output_dir", symbols_dir.path)
```

```suggestion
    args.add("--output_dir", symbols_dir)
```

we can let the `args` expand the file path at execution time, instead of doing so during analysis

---

Does that work with directories?

---

> Error in add: Cannot add directories to Args#add since they may expand to multiple values. Either use Args#add_all (if you want expansion) or args.add(directory.path) (if you do not).

---

seems to work with a little trick.